src/pages/brand/ContentManager/EditDrawer.tsx

```
@@ -0,0 +1,586 @@
+
+    switch (type) {
+      case 'bold':
+        newText = `${before}**${selectedText || 'bold text'}**${after}`
+        cursorPos = selectedText ? end + 4 : start + 2
+        break
+      case 'italic':
+        newText = `${before}*${selectedText || 'italic text'}*${after}`
+        cursorPos = selectedText ? end + 2 : start + 1
+        break
+      case 'link':
+        const url = prompt('Enter URL:')
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)